src/components/TaskInput.jsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from "react";
+import ToDoContext from "../context/ToDoContext";
+import { AiFillPlusCircle } from "react-icons/ai";
+const TaskInput = () => {
+  const { toDo, handleOnChange, task, handleTaskAdd } = useContext(ToDoContext);
+  console.log(toDo);
```

Comment:
You should remove this line, maybe you only need them in the develop tool testing process! so it's better to remove it now.